src/Restivus/Extensions.cs

```diff
@@ -26,7 +30,11 @@ public static Uri FilterQueryParams(this Uri uri, params string[] keys)
             {
                 foreach (var key in keys)
                 {
+#if NETSTANDARD13
+                    if (queryParams.ContainsKey(key))
+#elif NET4_5
                     if (queryParams.AllKeys.Contains(key))
+#endif
```

Comment:
Wondering if we could go with a slight style change here in order to involve as few as possible language constructs in this preprocessor conditional.

I'm thinking something like 

```cs
// ...
foreach (var key in keys)
{
    var shouldFilter =
#if NETSTANDARD13
        queryParams.ContainsKey(key);
#elif NET4_5
        queryParams.AllKeys.Contains(key);
#end

    if (shouldFilter) { queryParams[key] = "__FILTERED__"; }
}
// ...
```